android/app/src/main/res/values/strings.xml

```diff
@@ -212,4 +214,11 @@
     <string name="result_type">Result Type</string>
     <string name="error_create_notification">Error creating notification</string>
 
+    <!--Feed Strings-->
+    <string name="facebook_access_token" templateMergeStrategy="preserve" translatable="false">1786614831603417|DWqP48Lm77hBqCQDWVsq4lFixuo</string>
```

Comment:
Facebook's access token does not expire.Where to store the access token? Even the google play maps api key is stored in google_play.xml